### ISSUE TITLE
addMessage should add a message, not set the group to the message

### ISSUE THREAD

**Issue by aoloe** [Neoflow/FlashMessages#2]
Hi

This morning I've tried to use your library and I fear that I stumbled on a _simple_ bug: `Flash::addMessage()` should add the message not set the key to the message...

The current code is

```php
    /**
     * {@inheritDoc}
     */
    public function addMessage(string $key, $message): FlashInterface
    {
        if (!isset($this->next[$key])) {
            $this->next[$key] = [];
        }

        $this->next[$key] = $message;

        return $this;
    }
```

but adding the message should imo be:

```php
        $this->next[$key][] = $message;
```

Since you did define the return type of `getMessages()` php gave me a nice error instead of simply returning the wrong data : - ).

I will keep on testing your library, but, for now, it seems to do its job!

**Comment by Alpine418** [comment]
I will deep dive into your issue in the next few hours. Thanks for submitting the bug.

**Comment by aoloe** [comment]
Thanks : - )

**Comment by Alpine418** [comment]
Fixed the bug.  Currently I do not use the lib. So thanks again for your issue!

**Comment by Alpine418** [comment]
Please update the lib to the latest released version 1.0.1 to get the fix.

**Comment by aoloe** [comment]
updated, tested, it works!

merssi
